edgedb/base_con.py

```diff
@@ -101,6 +102,7 @@ def add_log_listener(
             **connection**: a Connection the callback is registered with;
             **message**: the `edgedb.EdgeDBMessage` message.
         """
+        # TODO(tailhook)
```

Comment:
Can you clarify this and the following TODOs?

Reply:
No need for `is_closed` check. Removed the check and TODO.